src/components/labelPrinting/LabelPrintingForm.vue

```diff
@@ -250,15 +255,19 @@ const onReset = () => {
   })
 }
 
-// fetch printers
-// if no printers are in the store, fetch them
+// if no printers or workflows in the store, fetch them
 // if there are printers in the store, return success prevents error in DataFetcher
 // @returns {Promise} - Promise
-const fetchPrinters = async () => {
-  if (printingStore.printers().length === 0) {
-    return await printingStore.fetchPrinters()
-  } else {
-    return { success: true }
+const fetchResources = async () => {
```

Comment:
I'm not sure this is the ideal way to deal with thi. @BenTopping can you have a look. From what I remember we should use the DataFetcher. Doing it this way could cause issues. Also we are always returning true whatever.